### ISSUE TITLE
Can't connect

### ISSUE THREAD

**Issue by boi1216** [InfinityGamers/GoProxy#4]
I'm not able to connect, the last message in console is
[33m[2018-08-10 09:08:06][T/4][Log/INFO]: [37mListening to Server address: 107.150.59.74:2000
Tried more servers, still not working

**Comment by WetABQ** [comment]
I have same issue

**Comment by xBeastMode** [comment]
Hi, does the MOTD show in-game? If not then it most likely means the server is encrypted. If not that it's probably some proxy blocker, which I don't know how to bypass.

**Comment by boi1216** [comment]
no motd does not show in game also the server is not encrypted it's simple pocketmine server

**Comment by WetABQ** [comment]
yes,no motd,That's just the server I used for testing, and it's not encrypted.@xBeastMode

**Comment by xBeastMode** [comment]
How are you sure it's not encrypted? Is it your own server? If so, run /version and post the output here and make sure ```network.enable-encryption``` is not set to true in pocketmine.yml. Also please note it could as well be a proxy blocker like I said

**Comment by WetABQ** [comment]
The server is on my local, I am using Nukkit, Nukkit does not have any encryption options.@xBeastMode

**Comment by boi1216** [comment]
Same. The server has no encryption, I am able to connect to the server though mcpeproxy.tk, but here it's not working..  pretty sure it's problem in the released exe file

**Comment by xBeastMode** [comment]
I have no idea then, tried to fix this but nothing has worked, it's probably just the fact that you're connecting to localhost. ```Local Client -> Local Proxy -> Local Server``` might not work but you can try ```Local Client  -> External Proxy -> Local Server``` if you have a relative who can help you run the proxy on their computer, or you can try it on a VPS if you have one

**Comment by WetABQ** [comment]
My vps is running on Ubuntu, it can't run exe files, I can't package Go language project @xBeastMode

**Comment by WetABQ** [comment]
I just tried to use FRP intranet to break through my local server, that is, local->FRP extranet server->local proxy->FRP extranet server->local server, it still does not work. @xBeastMode

**Comment by xBeastMode** [comment]
Hmmm. Well I guess there's nothing I can do, the connection is made how it's supposed to and there's nothing else I can change in the code that would make it work. I will try to make a runnable for linux but I lost the source code with the cheats somewhere.

**Comment by WetABQ** [comment]
Can you rewrite the project using php or java?
This is a good project

**Comment by xBeastMode** [comment]
This was only an experimental project, I never planned to release it but did anyway. I also don't plan on rewriting it. 😛

**Comment by WetABQ** [comment]
Then can you fix this problem that cannot be connected? @xBeastMode
